package.json

```diff
@@ -1412,7 +1408,7 @@
         "order": 9,
         "properties": {
           "dotnet.preferCSharpExtension": {
-            "scope": "resource",
+            "scope": "machine-overridable",
```

Comment:
I wonder if this should be "window".

scope ↓ / target → | user | remote | workspace | folder
-- | -- | -- | -- | --
application | ✔️ | ✖️ | ✖️ | ✖️
machine | ✖️ | ✔️ | ✖️ | ✖️
window | ✔️ | ✔️ | ✔️ | ✖️
resource | ✔️ | ✔️ | ✔️ | ✔️

Reply:
You're probably right - since it isn't possible to have a different value for this for different folders (theres only one extension active at a time).